include/GFp/base.h

```diff
@@ -87,6 +87,9 @@
 #elif defined(__mips__) && defined(__LP64__)
 #define OPENSSL_64_BIT
 #define OPENSSL_MIPS64
+#elif defined(__wasm32__)
+#define OPENSSL_32_BIT
+#define OPENSSL_X86_64
```

Comment:
It definitely isn't right to define `OPENSSL_X86_64` for WASI.

We already have:
```
#elif defined(__wasm__)
#define OPENSSL_32_BIT
```

Is that not good enough? I think we can just revert this chunk.